quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/Environment.java

```diff
@@ -37,6 +37,7 @@
 
 public final class Environment {
 
+    public static final String KC_CONFIG_REBUILD_CHECK = "kc.config.build-and-exit";
```

Comment:
Should this match the naming with the methods?
```suggestion
    public static final String KC_CONFIG_REBUILD_CHECK = "kc.config.rebuild-check";
```